lib/node/server.ml

```diff
@@ -128,7 +131,7 @@ let _print_logs node =
    3. Run the disseminator, this includes actually sending messages to be
       disseminated across the network.
    4. Wait 0.001 seconds before restarting the procedure. *)
-let rec run node preprocessor msg_handler =
+let rec run node preprocessor (msg_handler: Message.t -> bytes option * bytes option ) =
```

Comment:
Shouldn't it be `(bytes * bytes option) option`? Since you can't have a payload_signature without a payload?